samples/highcharts/demo/neural-network/demo.js

```diff
@@ -0,0 +1,101 @@
+const layers = [1, 6, 6, 6, 2];
+const activations = ['tanh', 'tanh', 'ReLU', 'tanh', 'sigmoid'];
+
+const categories = Array.from({ length: layers.length }, (_, i) => {
+    if (i === 0) {
+        return `Input Layer (${activations[i]})`;
+    }
+
+    if (i === layers.length - 1) {
+        return `Output Layer (${activations[i]})`;
+    }
+
+    return `Hidden Layer #${i} (${activations[i]})`;
+});
+
+function generateData() {
+    if (layers.length === 0) {
+        return [];
+    }
+
+    const data = [];
+
+    function generate(currentIndices) {
+        if (currentIndices.length === layers.length) {
+            data.push({
+                data: [...currentIndices]
+            });
+            return;
+        }
+
+        const dimensionIndex = currentIndices.length;
+        for (let i = 0; i < layers[dimensionIndex]; i++) {
+            generate([...currentIndices, i]);
+        }
+    }
+
+    generate([]);
+    return data;
+}
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'line',
+        parallelCoordinates: true,
+        inverted: true
+    },
+    title: {
+        text: 'Visualizing a neural network with Highcharts'
+    },
+    subtitle: {
+        text: 'You can use the parallel-coordinates module ' +
+            'to visualize a neural network.'
+    },
+    tooltip: {
+        formatter() {
```

Comment:
Could you replace formatter with `format`? You can store extra info per series/point/axis.